validator/keymanager/remote-web3signer/internal/client.go

```diff
@@ -151,6 +150,10 @@ func (client *ApiClient) doRequest(ctx context.Context, httpMethod, fullPath str
 	}
 	req.Header.Set("Content-Type", "application/json")
 
+	requestDump, err := httputil.DumpRequestOut(req, true)
```

Comment:
before merging this in, any considerations on performance here? to keep a copy of the request/body before having to go through the dump.My  only concern is dumping causing lag if there are no errors